### ISSUE TITLE
Question about fine tuning

### ISSUE THREAD

**Issue by CheukHinHoJerry** [yuxumin/PoinTr#56]
Sorry for disturbing. I have some problem on fine tuning with pretrained parameters.

I tried to fine tune the ShapeNet55 PoinTr on a specific "Bench" subset of ShapeNet. What I do it extracting the subset data, use the pretrained PoinTr model that you have provided and rename it as weight.pth. Then run:

bash ./scripts/train.sh 0     --config ./cfgs/ShapeNet55_models/PoinTr.yaml     --exp_name example --start_ckpts ./weight.pth

however, the loss does not goes down and the value of prediction after training ranges from ~10e-5 to 10e5, which is non-sense.

Did I do anything wrong? If not, is there any other method that I can fine tune the pretrained model on a specific dataset?

**Comment by yuxumin** [comment]
Hi, how do you construct the "bench" subset? And if all the problems come from the training phase, i think the discussion (https://github.com/yuxumin/PoinTr/issues/51) will help.

**Comment by CheukHinHoJerry** [comment]
I construct the bench subset by getting the .npy files with corresponding object id from the training/testing data that download from google drive. Do you think it is fine?

I set the batch size to 16 since I am training on a local computer with GTX 1050Ti. I go through the previous discussion and it seems that the LR has to be changed too? Do you think multiplying the original LR by (original BS)/(new BS) works?

**Comment by yuxumin** [comment]
Have you taken a look at the tensorboard? I guess you encounter the problem as the issue i mentioned above (losses blow up).
